packages/sn-document-viewer-react/test/DocumentTitlePager.test.tsx

```diff
@@ -0,0 +1,55 @@
+import TextField from '@material-ui/core/TextField'
+import Typography from '@material-ui/core/Typography'
+import { sleepAsync } from '@sensenet/client-utils'
+import { shallow } from 'enzyme'
+import React from 'react'
+import { DocumentTitlePagerComponent } from '../src/components/document-widgets/DocumentTitlePager'
+
+describe('DocumentTitlePager component', () => {
+  it('Should render the unfocused state without crashing', () => {
+    const wrapper = shallow(
+      <DocumentTitlePagerComponent
+        activePages={[1]}
+        documentName="TestDocument"
+        gotoPage="goto"
+        pageCount={10}
+        setActivePages={jest.fn()}
+      />,
+    )
+    expect(wrapper).toMatchSnapshot()
+  })
+
+  it('Should render the focused state without crashing', async () => {
+    const wrapper = shallow(
+      <DocumentTitlePagerComponent
+        activePages={[1]}
+        documentName="TestDocument"
+        gotoPage="goto"
+        pageCount={10}
+        setActivePages={jest.fn()}
+      />,
+    )
+    wrapper.find(Typography).simulate('click')
+    await sleepAsync()
```

Comment:
I think the sleep is not neccessary but we should rather make an assertion if a child element exists.
```suggestion
    expect(wrapper.find(TextField)).toBeTruthy()
```